src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj

```diff
@@ -7,10 +7,12 @@
     <Version>1.11.1-prerelease</Version>
     <Title>Datadog APM - ClrProfiler</Title>
     <Description>OBSOLETE. This package is deprecated and exists only for backwards compatibility.
```

Comment:
If this package is obsolete, should we continue releasing new versions?

Reply:
Mostly just I want to update the description text. It's caused some confusion among users. For example, see #609, #619, and some conversations on public Slack.

But we may need to keep updating it anyway to keep the version in sync with the installers for users who haven't realized that they should remove the package.

Reply:
If they are upgrading, wouldn't that be when we want them to stop including it?

Reply:
Ideally, yes, but we don't have a way from stopping a user from upgrading blindly without reading the new package description. And if a user updates the installer and keep the older NuGet package (because there's no newer version), then non-matching versions causes issues.

Keep in mind that users were using this packages for 3 different use cases:
- manual instrumentation (because it includes `Datadog.Trace`)
- ASP.NET integration (before it moves to `Datadog.Trace.AspNet`)
- auto-instrumentation on .NET Core (before the assemblies were moved into the installer)

I think it's little effort for us to keep bumping the version here for now so the package keeps working for users who are using it. We can drop it entirely when we make other big breaking changes (2.0?).